website/sidebars.js

```diff
@@ -57,6 +57,7 @@ const sidebars = {
                 'monitoring',
                 'kaito-oom-prevention',
                 'kaito-on-byo-gpu-nodes',
+                'headlamp-kaito',
```

Comment:
Could you move headlamp-kaito to Features sections instead of Operations section?